scripts/gha/integration_testing/test_validation.py

```diff
@@ -233,8 +233,8 @@ def summarize_test_results(tests, platform, summary_dir, file_name="summary.log"
       summary_json["failures"][testapp]["raw_result_link"] = test.raw_result_link
     failed_tests = re.findall(r"\[  FAILED  \] (.+)[.](.+)", results.summary)
     for failed_test in failed_tests:
-      failed_test = failed_test[1]
-      pattern = fr'\[ RUN      \] (.+)[.]{failed_test}(.*?)\[  FAILED  \] (.+)[.]{failed_test}'
+      failed_test = failed_test[0] + "." + failed_test[1]
+      pattern = fr'\[ RUN      \] {failed_test}(.*?)\[  FAILED  \] {failed_test}'
```

Comment:
nit: would it better to match ` *RUN *` (space-star-RUN-space-star) in the regexp, and similar with FAILED?